src/store/site.ts

```diff
@@ -1,5 +1,15 @@
-import { atomWithStorage } from "jotai/utils"
+import { atomWithStorage, createJSONStorage } from "jotai/utils"
 
 import type { SiteKey } from "../types/site"
 
-export const siteAtom = atomWithStorage<SiteKey>("site", "stitch")
+export const SITE_KEY = "site"
+export const DEFAULT_SITE: SiteKey = "stitch"
+
+export const siteAtom = atomWithStorage<SiteKey>(
+  SITE_KEY,
+  DEFAULT_SITE,
+  createJSONStorage(),
+  {
+    getOnInit: true,
```

Comment:
without `getOnInit: true`, we get a flash of the default theme - this improves performance.